recipes/newrelic/infrastructure/agent-control/debian.yml

```diff
@@ -478,10 +478,13 @@ install:
         - |
           if [ "{{.NEW_RELIC_REGION}}" = "STAGING" ]; then
             sed -i 's/\(endpoint: https:\/\/opamp.\).*/\1'"staging-service.newrelic.com\/v1\/opamp"'/' /etc/newrelic-agent-control/config.yaml
+            sed -i '/^fleet_control:/a \  signature_validation:\n    public_key_server_url: https:\/\/staging-publickeys.newrelic.com\/r\/blob-management\/GLOBAL\/AgentConfiguration\/jwks.json' /etc/newrelic-agent-control/config.yaml
```

Comment:
nit, it would be a bit more clear if we replace a placeholder like <PLACEHOLDER> with the url , perhaps is it hard to do the change now + it would be nice to change all this script to use some more clear tool like yq